### ISSUE TITLE
TYPO3 13 support

### ISSUE THREAD

**Issue by the-andyman** [Hauer-Heinrich/hh_video_extender#41]
Is a version for TYPO3 13 planned, or perhaps already in the works?
Thanks a lot!

**Comment by Teisi** [comment]
Yes, it's planned, probably this week or next - depending on how much time I can carve out for it.

**Comment by the-andyman** [comment]
Hello, I wanted to ask carefully if there is already progress with a TYPO3 13 compatible version?

**Comment by Teisi** [comment]
Version 0.3.0 should work with TYPO3 13 - could you also verify this and let me know? :)

**Comment by bh-teufels** [comment]
{settings.hhvideoextender.includeCss} does not work unter TYPO3 v13 it is NULL
although plugin.tx_hhvideoextender. includeCss = 1 is set by default

**Comment by Teisi** [comment]
@bh-teufels should be fixed
